rules-reviewed/openjdk11/openjdk8/java-removals.windup.xml

```diff
@@ -105,5 +105,29 @@
                <matches pattern="(checkAwtEventQueueAccess|checkSystemClipboardAccess|checkTopLevelWindow)"/>
             </where>
         </rule>
+        <rule id="java-removals-00110">
+            <when>
+                <javaclass references="sun.misc.Unsafe.{method-name}({*})"/>
+            </when>
+            <perform>
+                <hint title="Methods in `sun.misc.Unsafe` have been removed" effort="2" category-id="mandatory">
+                    <message>
+                        The `sun.misc.Unsafe` API is scheduled for removal in the long term, and some of its methods have been removed. In short term, please consider the following:
+
+                        - The `get` and `put` methods have equivalents that take long offsets (OpenJDK 8's version of the removed methods just casts the int offsets and used the long versions).
+                        - `staticFieldOffset(Field)` and `ObjectFieldOffset(Field)` should replace `fieldOffset(Field)` (which the 8 version delegated to already).
```

Comment:
I understand this message comes from the Ticket itself. But usually, when there are docs mentioning replacements it follows the pattern: `A should be replaced by B` this way it is easy to understand the change `source => target`. The message "staticFieldOffset(Field) and ObjectFieldOffset(Field) should replace fieldOffset(Field)" follows the oposite pattern: `B should replace A`, the pattern is `target => source`. I would suggest to reorder the text. However, I acknowledge English is not my native language so this is just a suggestion and not a required change.

Reply:
Will do :+1: